mirrord-protocol/src/lib.rs

```diff
@@ -1,9 +1,30 @@
 #![feature(const_trait_impl)]
 #![feature(io_error_more)]
 #![feature(core_ffi_c)]
+#![feature(hash_drain_filter)]
```

Comment:
Where are you using this?

Reply:
Same question re OnceCell

Reply:
Good catch, was using this while experimenting, and missed removing in the clean up.